spring-boot-project/spring-boot-docs/src/docs/asciidoc/io/rest-client.adoc

```diff
@@ -39,7 +39,7 @@ include::{docs-java}/io/restclient/resttemplate/customization/MyRestTemplateCust
 ----
 
 Finally, you can also create your own `RestTemplateBuilder` bean.
-To prevent switching off the auto-configuration of a `RestTemplateBuilder` and prevent any `RestTemplateCustomizer` beans from being used, make sure to configure your custom instance with a `RestTemplateBuilderConfigurer`.
+To prevent switching off the auto-configuration of a `RestTemplateBuilder` and prevent any `RestTemplateCustomizer` beans from not being used, make sure to configure your custom instance with a `RestTemplateBuilderConfigurer`.
```

Comment:
"prevent from not being used" reads as a double negation to me, I am not sure what you mean by this change.

If you want to create your own `RestTemplateBuilder`, you should use `RestTemplateBuilderConfigurer` as an input (as the next example shows).  This will make sure that customizers are indeed called.